app/Middleware/Adminrequest.ts

```diff
@@ -0,0 +1,15 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Database from '@ioc:Adonis/Lucid/Database'
+
+export default class Adminrequest {
+  public async handle({auth , response}: HttpContextContract, next: () => Promise<void>) {
+    // code for middleware goes here. ABOVE THE NEXT CALL
+    const role = await Database. from('roles').where('user_id' , auth.user.id).select('role')
```

Comment:
why is there a space in query? const role = await Database.` `from('roles').where('user_id' , auth.user.id).select('role')